pkg/auth.go

```diff
@@ -132,7 +139,7 @@ func (auth *Auth) Configure() (err error) {
 // CreateContractSession creates a session based on an IRMA contract. This allows the user to permit the application to
 // use the Nuts Network in its name. The user can limit the application in time and scope. By signing it with IRMA other
 // nodes in the network can verify the validity of the contract.
-func (auth *Auth) CreateContractSession(sessionRequest CreateSessionRequest, actingParty string) (*CreateSessionResult, error) {
+func (auth *Auth) CreateContractSession(sessionRequest CreateSessionRequest) (*CreateSessionResult, error) {
```

Comment:
After this PR, is actingParty still used anywhere? If not, link this PR to an issue so we can track it and update the manuals

Reply:
As far as I know there is currenly nog much documented about all the different types of logincontracts. I created this issue: https://github.com/nuts-foundation/nuts-documentation/issues/39

Reply:
I think actingPartyCN can be removed in favor of the global 'identity' parameter; when you need a (X500?) name of the party it can be looked up in the registry using the identity parameter.